.ci/deploy-windows.sh

```diff
@@ -20,5 +20,5 @@ sha256sum "$BUILD" | awk '{ print $1 }' | tee "$BUILD.sha256"
 echo "$(cat "$BUILD.sha256");$(stat -c %s "$BUILD")B" > GitHubReleaseMessage.txt
 
 # Move files to publishing directory
-mv -- "$BUILD" "$ARTIFACT_DIR"
-mv -- "$BUILD.sha256" "$ARTIFACT_DIR"
+cp -- "$BUILD" "$ARTIFACT_DIR"
```

Comment:
why did this change?

Reply:
Because the upload script for releases uploads from a different directory, while for Cirrus artifacts we can simply have it pick the 7z from the directory (this avoids subdirs in the cirrus artifact page).